src/cephadm/cephadm

```diff
@@ -2217,12 +2219,16 @@ def get_config_and_keyring(ctx):
         keyring = d.get('keyring')
 
     if 'config' in ctx and ctx.config:
+        if not os.path.isfile(ctx.config):
+            raise Error('config file %s not found' % ctx.config)
         with open(ctx.config, 'r') as f:
             config = f.read()
 
     if 'key' in ctx and ctx.key:
         keyring = '[%s]\n\tkey = %s\n' % (ctx.name, ctx.key)
     elif 'keyring' in ctx and ctx.keyring:
+        if not os.path.isfile(ctx.keyring):
+            raise Error('keyring file %s not found' % ctx.keyring)
```

Comment:
Again, the next line calls `open()` which will raise `FileNotFoundError`.

Reply:
same as above